frontend/src/components/MapView/Download/index.tsx

```diff
@@ -71,8 +71,8 @@ function Download({ classes }: DownloadProps) {
     ({ id }) => id === 'adamts_buffers',
   );
 
-  const isShowingFloodData = selectedLayers.some(({ id }) =>
-    id.includes('hydra_'),
+  const isShowingFloodData = selectedLayers.some(
```

Comment:
Can you add a TODO above this to explore moving this to a configuration option in layers.json?